src/Body.ts

```diff
@@ -9,6 +10,8 @@ import * as bodyClasses from './styles/body.m.css';
 
 export const BodyBase = ThemeableMixin(RegistryMixin(WidgetBase));
 
+export type BodyInterface = WidgetBaseConstructor<BodyProperties>;
```

Comment:
Wouldn't this be better named `BodyConstructor`?  We have avoided naming interfaces/types with the name `Interface` in the name.